src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+                    'model_type': 'reanalysis/forecast'
+                }
+            )
+            sensors.append(sensor)
+            
+        return sensors
+        
+    async def _load_data_for_period(
+        self,
+        start_date: datetime,
+        end_date: datetime,
+        use_forecast: bool = False
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)